dapps/marketplace/src/components/Reviews.js

```diff
@@ -184,7 +186,7 @@ export default class Reviews extends Component {
                   }}
                 >
                   {loading ? (
-                    <fbt desc="reviews.loadingMore">Loading More...</fbt>
```

Comment:
It may be appropriate to run the translation commands to remove these now unused string instances. 🤔

Reply:
Ok I'll look into that.

Reply:
I've updated the translations